Looport/Millennium-Falcon
# Create messenger service

# Description


In this PR, I added a messenger service and the ability to listen to passport user registration events and save users in to messenger database. The messenger copies most of the logic from the passport service.

This PR also includes CI/CD configuration.

## What Changed


— Added the messenger service
— Removed global jwt module imports

## How To Test


— Run messenger and passport service
— Create user in the passport service
— Check messenger db where user should be created

# Checklist
- [x] I followed all possible cases of the implemented _user flow_
